provider/acme/provider.go

```diff
@@ -368,6 +389,53 @@ func (p *Provider) resolveCertificate(domain types.Domain, domainFromConfigurati
 	return certificate, nil
 }
 
+func (p *Provider) useBackOffToObtainCertificate(domains []string) bool {
+	// Check if we can use a backoff only if we use the DNS Challenge and if is there are at least 2 domains to check
+	if p.DNSChallenge != nil && len(domains) > 1 {
+		rootDomain := ""
+		for i := 0; i < len(domains); i++ {
```

Comment:
`for i := range domains` instead?

![download](https://user-images.githubusercontent.com/6207234/43407257-825b8d84-941e-11e8-9477-b69d739c66c4.jpeg)

Reply:
@emilevauge I used the old way to iterate on a slice in the way to use only one `loop` statement for the two operations I have to to on the slice.

I was not aware about the possible issues you mentionned, I am going to change the code.